test/integration/submit_pledge_test.rb

```diff
@@ -6,6 +6,9 @@ class SubmitPledgeTest < ActionDispatch::IntegrationTest
     @user = create :user
     @patient = create :patient
```

Comment:
FactoryGirl lets you create stuff and assign attributes in one line -- if you do `@patient = create :patient, clinic_name: 'Nice Clinic', appointment_date: DateTime.now + 14` and `@pregnancy = create :pregnancy, patient: @patient, dcaf_soft_pledge: 500` it'll create the objects and set those attributes while it does it. 

I got the test to pass using this! Maybe it wasn't saving as written?